doc/examples/anim13/anim13.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+#
+# Movie of seismic waveforms from 2020-10-06 Alaska earthquake
+# Epicenter 54.844N 159.842W
+# Depth 31.0 km
+# UTC time 2020-10-06 05:54:50
+# Recorded at station Dolgoi Island, Mount Dutton, Alaska
+#
+# USGS event page: https://earthquake.usgs.gov/earthquakes/eventpage/us6000c5zm/executive
+#
+# DEM:  @earth_relief_06m
+# Data: @waveform_AV.DOL.txt
+# 
+# The data file @waveform_AV.DOL.txt contains 4 columns: 
+# UTC times, and three-component movement in east-west, north-south and up-down directions
+#
+# The finished movie is available in our YouTube channel as well:
+# https://youtu.be/eq_-2nYYf-s
+# The 66-sec, 1561 frame movie took ~15 minutes to render on a 24-core MacPro 2013.
+
+# 0. Create an include file with key parameters
+cat << EOF > inc.sh
+TIME_RANGE=2020-10-06T05:55:10/2020-10-06T05:56:15
+PLOT_DOMAIN=2020-10-06T05:55:10/2020-10-06T05:56:15/-2.6/2.8
+DPI=200i
+SPLINE=linear
+EOF
+# 1. Create a dense point-file from the waveform data
+cat << EOF > pre.sh
+gmt begin
+	# 1a. Create dense point-files from the waveform data components so we can animate the lines
+	gmt events @waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,1+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > E.txt
+	gmt events @waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,2+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > N.txt
+	gmt events @waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,3+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > Z.txt
+	# 1b. Setup the desired output times (24 frames per second)
+	gmt math -T\${TIME_RANGE}/24+i -o0 T --TIME_UNIT=s --FORMAT_CLOCK_OUT=hh:mm:ss.xxxxx = times.txt
+	# 1c. Lay down the static part of the subplot
+	gmt subplot begin 3x1 -Fs18.25c/3.5c -A -M0 -R\${PLOT_DOMAIN} -X5.5c -Y0.75c
+		gmt subplot set 0 -A"Z"
+		gmt plot @waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWsrt -W0.5p,darkgray -i0,3+s1e-6
+		gmt subplot set 1 -A"N"
+		gmt plot @waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWsrt -W0.5p,darkgray -i0,2+s1e-6
+		gmt subplot set 2 -A"E"
+		gmt plot @waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWSrt -W0.5p,darkgray -i0,1+s1e-6
+	gmt subplot end
+	# 1d. Create location map and metadata
+	gmt text -R0/4.5/0/13 -Jx1c -F+j+f -B0 -X-5.25c -Y-0.5c <<- EOF
+	2.25 12.5 CM 18p 2020-10-06
+	2.25 11.7 CM 9p SE of Sand Point, Alaska
+	2.25 11 CM 10p 54.844@.N, 159.842@.W
+	2.25 10.5 CM 10p 31.0 km, Mw 6.0
+	2.25 10.0 CM 10p 2020-10-06 05:54:50 UTC
+	EOF
+	gmt grdimage -R190W/130W/30/75 @earth_relief_06m -Ba30+f -JM3.5c -I+d -Cterra --MAP_FRAME_TYPE=plain -X0.75c -Y0.5c -BWNbr --MAP_ANNOT_OBLIQUE=lat_parallel --FONT_ANNOT_PRIMARY=9p
+	echo 161.86W 55.15N | gmt plot -St0.2c -GBlack
+	echo 159.842W 54.844N | gmt plot -Sa0.25c -Gred -Wfaint
+	gmt meca -Sm2.5c -M -G110/168/255 -T0 -W0.5p -Yh+0.5c <<- EOF
+	-159.70 54.81 39 0.82 -0.68 -0.14 0.63 0.34 -0.40 25
```

Comment:
@seisman, why was the moment tensor coordinates different than the event coordinates? These may need to be updated based on https://earthquake.usgs.gov/earthquakes/eventpage/us6000c5zm/moment-tensor?source=us&code=us_6000c5zm_mww, but I want to check with you about proper coordinates first.

Reply:
I think I used the focal mechanism from the [GCMT solution](https://www.globalcmt.org/cgi-bin/globalcmt-cgi-bin/CMT5/form?itype=ymd&yr=2020&mo=10&day=6&oyr=1976&omo=1&oday=1&jyr=1976&jday=1&ojyr=1976&ojday=1&otype=nd&nday=1&lmw=5.8&umw=10&lms=0&ums=10&lmb=0&umb=10&llat=-90&ulat=90&llon=-180&ulon=180&lhd=0&uhd=60&lts=-9999&uts=9999&lpe1=0&upe1=90&lpe2=0&upe2=90&list=0). 

I agree that it would be better to use consistent coordinates for the event and beachball, but note that the USGS page provides focal mechanism in a different format (strike, dip, rake), so we need to change `-Sm2.5c` to `-Sc2.5c`.

Reply:
OK, I will add a suggestion using the USGS data and will confirm that it appears the same as their figure (I can't remember the last time I needed to plot a beachball - certainly for some class - so this is exciting)

Reply:
I couldn't get the fill to work as expected -  any recommendations here?
```
gmt begin
	gmt meca -R190W/130W/30/75 -JM3.5c -Sc2.5c -M -G110/168/255 -T0 -W0.5p -Yh+0.5c <<- EOF
-159.851 54.851 40.5 245 29 93 61 61 88 1.048 18
EOF
gmt end show
```

Reply:
I think you need to remove **-T0**.

> For double couple mechanisms, the -T option renders the beach ball transparent by drawing only the nodal planes and the circumference. For non-double couple mechanisms, -T0 option overlays best double couple transparently.

Reply:
> I think you need to remove **-T0**. It's documented, but I'm not sure if it makes sense or not.
> 
> > For double couple mechanisms, the -T option renders the beach ball transparent by drawing only the nodal planes and the circumference. For non-double couple mechanisms, -T0 option overlays best double couple transparently.

I can assume it makes sense to seismologists - just was not clear to me right away.